db/seeds.rb

```diff
@@ -43,6 +43,21 @@ class SeedAdminENVError < StandardError
   end
 end
 
+# This populates the 'regions' table for Swedish regions (aka counties),
+# as well as 'Sweden' and 'Online'.  This is used to specify the primary
+# region in which a company operates.
+#
+# This uses the 'city-state' gem for a list of regions (name and ISO code).
+# (That gem will also return a list of cities within region)
+#
+# If the regions table is not empty, do not run this code unless there
+# are also no companies, as the region ID's will be reset.
+if Region.exists?
```

Comment:
Region.exists? tests whether the table has nay data in it.  So, this code won't run if the seeding has already been done.  Since this is just a reference table seeding it once should suffice (although, we could always add records *to* the table after initial seeding.

In any case, I'll look at alternatives for doing this outside of the seed file.